merkle/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "merkletree"
-version = "0.4.2-alpha.0"
```

Comment:
BLOCKING: please undo, this change happens automatically when I run the release script